From 51b08eb375d0cede3b6430859c84227b80267402 Mon Sep 17 00:00:00 2001
From: Rai Harninder-B01044 <harninder.rai@freescale.com>
Date: Mon, 18 Apr 2016 15:39:11 +0800
Subject: [PATCH 001/124] arm64: Move LS2080A compilation under ARCH_LAYERSCAPE

commit d40c2fe65ebfbd88fd5e2eafc92df6a597c28136
[context adjustment]

ARCH_LAYERSCAPE is the common define for Layerscape SoC family
So, move the compilation of LS2085A device tree files under
this define

Signed-off-by: Rai Harninder-B01044 <harninder.rai@freescale.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/boot/dts/freescale/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 4f2de3e7..fc6fca3c 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -1,4 +1,4 @@
-dtb-$(CONFIG_ARCH_FSL_LS2085A) += fsl-ls2085a-simu.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls2085a-simu.dtb
  
 always		:= $(dtb-y)
 subdir-y	:= $(dts-dirs)
-- 
2.1.0.27.g96db324

